README.md

```diff
@@ -112,3 +143,11 @@ that service version can be released to a Harmony environment by updating the
 main Harmony Bamboo deployment project. Find the environment you wish to
 release the service version to and update the associated environment variable
 to update the semantic version tag at the end of the full Docker image name.
+
+## Get in touch:
+
+You can reach out to the maintainers of this repository via email:
+
+* david.p.auty@nasa.gov
+* matthew.savioe@colorado.edu
```

Comment:
```suggestion
* matthew.savoie@colorado.edu
```
I don't love email, but I should be able to ignore it.

Reply:
Gah! Sorry!